doc/generic/pgf/pgfmanual-en-gd-usage-pgf.tex

```diff
@@ -411,8 +411,8 @@ \subsubsection{Graph Drawing Scopes}
     commands |--|,  |..| (followed by Bézier coordinates), and |--cycle|.
 
     The parameter \meta{animations} contains algorithmically-generated
-    animation commands (calls to |\pgfanimateattribute|. The |whom| will be set
-    to |pgf@gd|.
+    animation commands (calls to |\pgf|\-|animate|\-|attribute|. The |whom|
```

Comment:
This will be problematic as it breaks the reverse command lookup.

Reply:
Currently short verbatim like `|\cmd|` is a thin wrap of `\verb|\cmd|` and won't create hyper target. Only those in `codeexample` and `command` env etc. will. So the reverse command lookup is not affected?

Reply:
Hmm, it may break search in PDF depending on PDF viewers. For example, Adobe Reader can still search the result for `\pgfanimateattribute`, but Skim can't.

Reply:
`\-` are totally avoided now, see https://github.com/pgf-tikz/pgf/pull/1210/commits/fccd2ed431eaa0f150d498ca641470cae11bb1eb.